superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -814,6 +858,25 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
         </>
       );
     }
+
+    // Import doesn't use db, so footer will not render in the if statement above
```

Comment:
Can you clarify what you mean with "Import doesn't use db"? Do you mean the `db` state?

Reply:
Yeah that's what I meant, it doesn't use db state. I clarified that in [`this commit`](https://github.com/apache/superset/pull/19314/commits/f1a7e8bab5e67b2a60da331745001914c1ef8e53) - do you think that's good or should I clarify further?